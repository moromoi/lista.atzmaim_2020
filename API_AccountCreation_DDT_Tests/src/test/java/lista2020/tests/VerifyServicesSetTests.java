package lista2020.tests;

import org.approvaltests.Approvals;
import org.testng.annotations.Test;

import java.util.HashSet;

import static org.testng.Assert.assertEquals;

public class VerifyServicesSetTests extends TestBase {

    @Test(priority = 1)
    public void getHolisticJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[1]"));

    }

    @Test(priority = 2)
    public void getConsultansJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[2]"));
    }

    @Test(priority = 3)
    public void getCosmeticsJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[3]"));
    }

    @Test(priority = 4)
    public void getHairSalonJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[4]"));
    }

    @Test(priority = 5)
    public void getInteriorJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[5]"));
    }

    @Test(priority = 6)
    public void getInstallersJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[6]"));
    }

    @Test(priority = 7)
    public void getMakeupJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[7]"));
    }

    @Test(priority = 8)
    public void getNailJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[8]"));
    }

    @Test(priority = 9)
    public void getMassageJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[9]"));
    }

    @Test(priority = 10)
    public void getMoversJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[10]"));
    }

    @Test(priority = 11)
    public void getNutritionistsJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[11]"));
    }

    @Test(priority = 12)
    public void getPestJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[12]"));
    }

    @Test(priority = 13)
    public void getTeachersJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[13]"));
    }

    @Test(priority = 14)
    public void getSalesJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[14]"));
    }

    @Test(priority = 15)
    public void getServiceProvidersJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[15]"));
    }

    @Test(priority = 16)
    public void getPersonalJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[16]"));
    }

    @Test(priority = 17)
    public void getTattooJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[17]"));
    }

    @Test(priority = 18)
    public void getTherapistsJSON() {
        Approvals.verify(app.serviceRestHelper().createAccountWithType("[18]"));
    }

    @Test(priority = 19)
    public void getAllTypeJSON() {
        for (int i = 0; i < 1; i++) {

            Approvals.verify(app.serviceRestHelper().createAccountWithType("[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18]"));
        }
    }

    @Test(priority = 20)
    public void withoutTypeJSON() {
        for (int i = 0; i < 1; i++) {
            Approvals.verify(app.serviceRestHelper().createAccountWithType("[]"));
        }
    }




    @Test(priority = 21)
    public void deleteAccount() {

        assertEquals(app.serviceRestHelper().deleteAccount(), new HashSet<>() );
    }


}