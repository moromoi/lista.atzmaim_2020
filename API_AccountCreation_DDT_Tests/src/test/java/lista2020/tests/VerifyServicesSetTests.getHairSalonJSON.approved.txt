[{"id":1,"name":"תספורת גבר","price":"50","duration":15,"color":"#50e3c1","category":{"name":"Hair Styling","id":1}},{"id":2,"name":"תספורת ילד","price":"50","duration":15,"color":"#a44bc2","category":{"name":"Hair Styling","id":1}},{"id":3,"name":"תספורת אישה","price":"150","duration":30,"color":"#a44bc2","category":{"name":"Hair Styling","id":1}},{"id":4,"name":"פן קצר","price":"80","duration":30,"color":"#a44bc2","category":{"name":"Hair Styling","id":1}},{"id":5,"name":"פן ארוך","price":"100","duration":45,"color":"#a44bc2","category":{"name":"Hair Styling","id":1}},{"id":6,"name":"פן כלול","price":"0","duration":15,"color":"#a44bc2","category":{"name":"Hair Styling","id":1}},{"id":7,"name":"פן בייביליס","price":"120","duration":30,"color":"#a44bc2","category":{"name":"Hair Styling","id":1}},{"id":8,"name":"תסרוקת ערב","price":"250","duration":60,"color":"#a44bc2","category":{"name":"Hair Styling","id":1}},{"id":9,"name":"תספורת אישה + פן","price":"180","duration":60,"color":"#a44bc2","category":{"name":"Hair Styling","id":1}},{"id":10,"name":"תספורת גבר + זקן","price":"70","duration":30,"color":"#a44bc2","category":{"name":"Hair Styling","id":1}},{"id":11,"name":"יעוץ החלקה","price":"0","duration":15,"color":"#efa5fe","category":{"name":"Hair Coloring","id":2}},{"id":12,"name":"צבע ראש מלא","price":"220","duration":15,"color":"#efa5fe","category":{"name":"Hair Coloring","id":2}},{"id":13,"name":"צבע שורש","price":"150","duration":15,"color":"#efa5fe","category":{"name":"Hair Coloring","id":2}},{"id":14,"name":"שטיפה","price":"150","duration":15,"color":"#efa5fe","category":{"name":"Hair Coloring","id":2}},{"id":15,"name":"גוונים ראש מלא","price":"500","duration":75,"color":"#efa5fe","category":{"name":"Hair Coloring","id":2}},{"id":16,"name":"גוונים חצי ראש","price":"300","duration":45,"color":"#efa5fe","category":{"name":"Hair Coloring","id":2}},{"id":17,"name":"אומברה","price":"600","duration":120,"color":"#efa5fe","category":{"name":"Hair Coloring","id":2}},{"id":18,"name":"בליאז'","price":"800","duration":120,"color":"#efa5fe","category":{"name":"Hair Coloring","id":2}},{"id":19,"name":"הבהרה","price":"150","duration":60,"color":"#efa5fe","category":{"name":"Hair Coloring","id":2}},{"id":20,"name":"פתיחת תלתלים","price":"250","duration":45,"color":"#efa5fe","category":{"name":"Hair Coloring","id":2}},{"id":21,"name":"החלקת קארטין","price":"600","duration":45,"color":"#fff581","category":{"name":"Hair Straightening","id":3}},{"id":22,"name":"החלקה אורגנית","price":"600","duration":45,"color":"#fff581","category":{"name":"Hair Straightening","id":3}},{"id":23,"name":"החלקה משולבת","price":"800","duration":120,"color":"#fff581","category":{"name":"Hair Straightening","id":3}},{"id":24,"name":"החלקה יפנית","price":"1200","duration":60,"color":"#fff581","category":{"name":"Hair Straightening","id":3}},{"id":25,"name":"מחליק \/ גיהוץ","price":"0","duration":90,"color":"#fff581","category":{"name":"Hair Straightening","id":3}}]